ner/src/main/java/edu/illinois/cs/cogcomp/ner/IO/ResourceUtilities.java

```diff
@@ -37,14 +37,7 @@ public static InputStream loadResource(String resourceFile) {
             // jar)
             if (!new File(file).exists()) {
                 logger.debug("Loading {} from classpath", resourceFile);
-                List<URL> list = IOUtils.lsResources(ResourceUtilities.class, resourceFile);
```

Comment:
We had discussed this problem of reading a particular Chinese gazetteer in the classpath. This method cannot read all lines, but the method in LineIO can. I am trying to re-produce this issue, but the original method works fine somehow.... I'll remove my changes after some more tests.